java/src/jni/h5util.c

```diff
@@ -1098,130 +1100,127 @@ h5str_sprintf(JNIEnv *env, h5str_t *out_str, hid_t container, hid_t tid, void *i
         }
 
         case H5T_REFERENCE: {
-            if (H5Tequal(tid, H5T_STD_REF)) {
+            /*if (H5Tequal(tid, H5T_STD_REF))*/ {
```

Comment:
Since it seems we don't need to compare to H5T_STD_REF anymore, might as well just delete these commented out sections everywhere and save ourselves the indentation mess.

Reply:
Agreed.